_data/rules.yml

```diff
@@ -538,6 +538,50 @@ groups:
           - name: percona/mongodb_exporter
             doc_url: https://github.com/percona/mongodb_exporter
             rules:
+              - name: MongoDB replication lag
+                description: Mongodb replication lag is more than 10s
+                query: 'avg(mongodb_mongod_replset_member_optime_date{state="PRIMARY"}) - avg(mongodb_mongod_replset_member_optime_date{state="SECONDARY"})  > 10'
+                severity: critical
+              - name: MongoDB replication headroom
+                description: MongoDB replication headroom is <= 0
+                query: '(avg(mongodb_mongod_replset_oplog_tail_timestamp - mongodb_mongod_replset_oplog_head_timestamp) - (avg(mongodb_mongod_replset_member_optime_date{state="PRIMARY"}) - avg(mongodb_mongod_replset_member_optime_date{state="SECONDARY"}))) <= 0'
+                severity: critical
+              - name: MongoDB replication Status 3
+                description: MongoDB Replication set member either perform startup self-checks, or transition from completing a rollback or resync
+                query: "mongodb_mongod_replset_member_state == 3"
+                severity: critical
+              - name: MongoDB replication Status 6
+                description: MongoDB Replication set member as seen from another member of the set, is not yet known
+                query: "mongodb_mongod_replset_member_state == 6"
+                severity: critical
+              - name: MongoDB replication Status 8
+                description: MongoDB Replication set member as seen from another member of the set, is unreachable
+                query: "mongodb_mongod_replset_member_state == 8"
+                severity: critical
```

Comment:
If it is unreachable, does it mean cluster is not operational? If cluster is operational, then critical severity is a bit too much.